apps/moose-cli-npm/package.json

```diff
@@ -23,9 +23,9 @@
     "typescript": "^4.9.5"
   },
   "optionalDependencies": {
-    "@514labs/moose-cli-darwin-arm64": "latest",
-    "@514labs/moose-cli-darwin-x64": "latest",
-    "@514labs/moose-cli-linux-arm64": "latest",
-    "@514labs/moose-cli-linux-x64": "latest"
+    "@514labs/moose-cli-darwin-arm64": "0.3.205",
```

Comment:
Is the version set elsewhere? I expect this file and values would be updated by the CI/CD process?

Reply:
It is! this is mostly for the dev setup.

The script that bumps those versions during the release process can be found here: https://github.com/514-labs/moose/blob/main/apps/moose-cli-npm/scripts/release-cli.sh